core/src/main/java/org/apache/iceberg/BaseMetastoreTableOperations.java

```diff
@@ -54,6 +54,10 @@
   protected BaseMetastoreTableOperations() {
   }
 
+  protected String fullName() {
```

Comment:
Could we rename this to `tableName` so it is more clear for implementations? It would also be good to add Javadoc.

Reply:
Renamed and added javadoc.
Could you please check them?

Thanks, Peter